### PR TITLE
add log retention as variable, add additional tags, reformat codes

### PR DESCRIPTION
**Pull request by rizkimhd**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

***

Fixes #0000

***

Release note for [CHANGELOG](https://github.com/traveloka/terraform-aws-bake-ami/blob/master/CHANGELOG.md):

```release-note
NOTES:

* Any Notes regarding your submitted PR, like breaking changes or else.

FEATURES:

* **New Source:** `aws_000_0000` ([#references_to_issue](./))

ENHANCEMENTS:

* feature: Add support for new version of AWS API

BUG FIXES:

* Prevent error from evil bugs
```

***

Output from `terraform plan` command from changes you propose.

```
$ terraform plan

```

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,16 +1,7 @@
...
-    Name          = "/aws/codebuild/${local.bake_project_name}"
-    ProductDomain = "${var.product_domain}"
-    Service       = "${var.service_name}"
-    Environment   = "management"
-    Description   = "LogGroup for ${var.service_name} Bake AMI"
-    ManagedBy     = "terraform"
-  }
+  name              = "/aws/codebuild/${local.bake_project_name}"
+  retention_in_days = var.log_retention_in_days
+  tags              = local.cwl_log_grop_tags
```
Comment (handojo1) [inline main.tf:4]: is the naming intended?

Comment (rizkimhd) [reply main.tf:4]: I think the previous one is using that naming (hence there should be no name changes in this PR), what do you think @handojo1?

Comment (handojo1) [reply main.tf:4]: I am referring to grop in `cwl_log_grop_tags`

Comment (rizkimhd) [reply main.tf:4]: Ah I didn't realize the typo... let me update it ya, thanks for noticing it

Comment (rizkimhd) [reply main.tf]: updated ya @handojo1

**Review by febryantonius — APPROVED** [review]

**Pull request merged**